nymea-app/ui/images/wallbox/wallbox.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This icon doesn't really fit with the style at all. There's already an ev-charger icon. That's not exactly fitting the style either and looks a bit old-fashioned but I'd say for now use that and I'll see if I can update that to look more like this one but still fitting with the rest of the style

Reply:
Resolved, Wallbox view now uses ev-charger.svg

Reply:
I've updated the ev-charger icon last night... if you rebase your code on latest master you should get that.